packaging_automation/tests/test_package_cloud_statistics_collector.py

```diff
@@ -18,44 +25,65 @@
 PACKAGE_CLOUD_ADMIN_API_TOKEN = os.getenv("PACKAGE_CLOUD_ADMIN_API_TOKEN")
 REPO = PackageCloudRepo.azure
 ORGANIZATION = PackageCloudOrganization.citusdata
-db_parameters = DbParams(user_name=DB_USER_NAME, password=DB_PASSWORD, host_and_port=DB_HOST_AND_PORT, db_name=DB_NAME)
+db_parameters = DbParams(
+    user_name=DB_USER_NAME,
+    password=DB_PASSWORD,
+    host_and_port=DB_HOST_AND_PORT,
+    db_name=DB_NAME,
+)
 # 7 Records are fetched for each package from package cloud. To check the record count, we need to multiply package
 # count with 7
 PACKAGE_SAVED_HISTORIC_RECORD_COUNT = 7
 
-PACKAGE_CLOUD_PARAMETERS = PackageCloudParams(admin_api_token=PACKAGE_CLOUD_ADMIN_API_TOKEN,
-                                              standard_api_token=PACKAGE_CLOUD_API_TOKEN, organization=ORGANIZATION,
-                                              repo_name=REPO)
+PACKAGE_CLOUD_PARAMETERS = PackageCloudParams(
+    admin_api_token=PACKAGE_CLOUD_ADMIN_API_TOKEN,
+    standard_api_token=PACKAGE_CLOUD_API_TOKEN,
+    organization=ORGANIZATION,
+    repo_name=REPO,
+)
 
 
 def test_fetch_and_save_package_cloud_stats():
     db = create_engine(db_connection_string(db_params=db_parameters, is_test=True))
-    db.execute(text(f'DROP TABLE IF EXISTS {PackageCloudDownloadStats.__tablename__}'))
+    conn = db.connect()
```

Comment:
After version update sqlalchemy api has changed therefore I needed to change the code to add connections